docs/start/getting-started/fragments/ios/setup.md

```diff
@@ -36,10 +36,10 @@ Close your Xcode project and reopen it using `./YOUR-PROJECT-NAME.xcworkspace` f
 In a terminal window, run the following command (for this app, accepting all defaults is OK) in your project folder (the folder that contains your `xcodeproj` file):
 
 ```bash
-$ amplify init        # accept defaults
+amplify init
 ```
 
-The `awsconfiguration.json` configuration file should be created in the root directory. 
+Accept the default values. The `awsconfiguration.json` configuration file should be created in the root directory. 
```

Comment:
user studies have shown that people don't read much outside of codeblocks. we should make these either bold or retain them in comments